apps/recovery-portal/src/main/webapp/self-registration-with-verification-confirm.jsp

```diff
@@ -56,7 +63,9 @@
 
 
     if (StringUtils.isBlank(username) || StringUtils.isBlank(confirmationKey)) {
```

Comment:
Let's remove the username blank check as it's always null.

Reply:
Fix in [340df84](https://github.com/wso2/identity-apps/pull/1409/commits/340df84bc2b702c73bac9f243d574f6ded65fd76)